tests/journey/test_federal_senate_dataset.py

```diff
@@ -0,0 +1,37 @@
+import os
+from tempfile import gettempdir
+from unittest import main, skipIf, TestCase
+
+
+from serenata_toolbox.federal_senate.federal_senate_dataset import FederalSenateDataset
+
+
+class TestJourneyFederalSenateDataset(TestCase):
+    def setUp(self):
+        self.path = gettempdir()
+        self.subject = FederalSenateDataset(self.path)
+
+    @skipIf(os.environ.get('RUN_INTEGRATION_TESTS') != '1',
+            'Skipping integration test')
+    def test_journey_federal_senate_dataset(self):
+        # fetch_saves_raw_files
+        self.subject.fetch()
+        federal_senate_csv_files = ['federal-senate-{}.csv'.format(year) for year in range(self.subject.FIRST_YEAR, self.subject.NEXT_YEAR)]
+        for federal_senate_csv_file in federal_senate_csv_files:
+            file_path = os.path.join(self.path, federal_senate_csv_file)
+            self.assertTrue(os.path.exists(file_path), "fetch_saves_raw_files")
```

Comment:
Also noted inconsistencies on the use of double quotes and single ones... It would be nice to stick to one of them 💅